app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadContract.kt

```diff
@@ -42,5 +42,7 @@ interface SellPayloadContract {
         fun cashOut()
         fun onTokenAmountChanged(newValue: String)
         fun onUserMaxClicked()
+        fun onStart()
+        fun onStop()
```

Comment:
Can you name this methods with more context? 
For example, `checkSellLock` or smth

Reply:
we can implement DefaultLifecycleObserver presenter contract, to have this methods as Default

Reply:
we don't need it, it's rare case